docs/hub/mlx-image.md

```diff
@@ -0,0 +1,97 @@
+# Using mlxim at Hugging Face
+
+[`mlxim`](https://github.com/riccardomusmeci/mlx-image) is an image models library built on Apple [MLX](https://github.com/ml-explore/mlx). It tries to replicate the great [timm](https://github.com/huggingface/pytorch-image-models) library from Ross Wightman, but for MLX models.
+
+
+## Exploring MLX on the Hub
+
+You can find `mlxim` models by filtering using the `mlxim` library name, like in [this query](https://huggingface.co/models?library=mlxim&sort=trending).
+There's also an open [mlx-vision](https://huggingface.co/mlx-vision) space for contributors converting and publishing weights for MLX format.
+
+Thanks to MLX Hugging Face Hub integration, you can load MLX models with a few lines of code. 
+
+## Installation
+
+```bash
+pip install mlx-image
+```
+
+## Models
+
+Model weights are available on the [`mlx-vision`](https://huggingface.co/mlx-vision) community on HuggingFace.
+
+To load a model with pre-trained weights:
+```python
+from mlxim.model import create_model
+
+# loading weights from HuggingFace (https://huggingface.co/mlx-vision/resnet18-mlxim)
+model = create_model("resnet18") # pretrained weights loaded from HF
+
+# loading weights from local file
+model = create_model("resnet18", weights="path/to/resnet18/model.safetensors")
+```
+
+To list all available models:
+
+```python
+from mlxim.model import list_models
+list_models()
+```
+> [!WARNING]
+> As of today (2024-03-08) mlx does not support `group` param for nn.Conv2d. Therefore, architectures such as `resnext`, `regnet` or `efficientnet` are not yet supported in `mlxim`.
+
+## ImageNet-1K Results
```

Comment:
```suggestion
## ImageNet-1K Results

```

Reply:
final minor nit: it's best to use newlines before/after headings, just in case :)